hw_2.py

```diff
@@ -0,0 +1,17 @@
+books = [
+    {
+        "name": "Lord of the rings",
+        'price': 700
+    },
+    {
+        'name': 'Fluent Python',
+        'price': 650
+    },
+    {
+        'name': 'Harry Potter',
+        'price': 1300
+    },
+]
+book = min(books, key=lambda x: x['price'])
+
+print("The cheapest book is '%s'. It costs %d grn" % (book['name'], book['price']))
```

Comment:
Please try to use `format` instead of `%` symbol for string formatting.

Read for details:
https://realpython.com/python-string-formatting/

Reply:
I fix it